portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -69,8 +73,13 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
             String email = (String) entity.getProperty("email");
             String comment = (String) entity.getProperty("comment");
             long timestamp = (long) entity.getProperty("timestamp");
-        
-            Comment userComment = new Comment(name, email, comment,timestamp,id);
+
+            Translate translate = TranslateOptions.getDefaultInstance().getService();
```

Comment:
You can put both the translation service, and especially the TranslateOption.targetLanguage, outside of the loop.

Reply:
done